src/api/spec/controllers/webui/session_controller_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe Webui::SessionController do
+  let!(:user) { create(:confirmed_user, login: 'tom') }
```

Comment:
use the lazy `let` instead, as this user is not used in `POST #create with deprecated password`.